src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -109,7 +109,8 @@ uct_gdr_copy_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
             switch (perf_attr->operation) {
             case UCT_EP_OP_GET_SHORT:
             case UCT_EP_OP_GET_ZCOPY:
-                perf_attr->bandwidth.shared = 250.0 * UCS_MBYTE;
+                perf_attr->bandwidth.dedicated = 250.0 * UCS_MBYTE;
```

Comment:
Assuming we do not have many processes per GPU, we may apply this change to the rest of ops here

Reply:
actually, put bw may be limited by PCIe bw that its shared. Since it's higher that get bw. need to test it..

Reply:
`ucx_perftest -t put_bw` showed that bandwidth of operation decreases when using more than one process per node, even if the processes use different GPUs. That is why I changed `shared` to `dedicated` only for get operation.

Reply:
ok
maybe we should move line 107 into each of the cases for `switch (perf_attr->operation)`?

Reply:
> ok maybe we should move line 107 into each of the cases for `switch (perf_attr->operation)`?

done